docs/Product-Strategy/Roadmap-and-Backlog.md

```diff
@@ -1,32 +1,32 @@
-_updated May 2021_
+_updated June 2021_
 # Product Roadmap
-Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1617994644349/6047bad9ed365b22986e52f5d9f01aac32286ae9) :lock:represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.
+Our [roadmap](https://app.mural.co/t/officeoffamilyassistance2744/m/officeoffamilyassistance2744/1617994644349/6047bad9ed365b22986e52f5d9f01aac32286ae9) :lock: represents our latest thinking about the order in which we’ll tackle the various pieces of the overarching problem.
 
 We consider ATO, Release 1 and Release 2 commitments that the team will deliver on at this time. There might be a few shifts in approach, timing or scope, but in general, these outcomes will be worked on by the team. Metrics for success will be added as they are defined for releases.
 
 Beyond that, we're still discovering and planning on what best serves our users. This doc will continue to be updated as we make decisions and scope releases. 
 
-## ATO  
+## ATO, _3 year ATO granted on May 20, 2021_  
 Value Delivered: Get approval for the authority to operate and create a production environment. 
 
 | Outcome | Status | 
 | -------- | ------- | 
 | User can log in using login.gov | Complete
 | Users with appropriate privileges can manage users | Complete
 | Users can upload data files by section and quarter | Complete
-| Users with appropriate privileges can download files that were previously uploaded     | In Progress
-|Create production environment | Will happen after ATO
+| Users with appropriate privileges can download files that were previously uploaded     | Complete
```

Comment:
which issue is this associated with?

Reply:
#416 I believe - but if that's incorrect, we can remove it for now.